lib/src/sql/statements/define/model.rs

```diff
@@ -30,31 +26,42 @@ impl fmt::Display for DefineModelStatement {
 	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
 		write!(f, "DEFINE MODEL ml::{}<{}>", self.name, self.version)?;
 		if let Some(comment) = self.comment.as_ref() {
-			write!(f, "COMMENT {}", comment)?;
-		}
-		if !self.permissions.is_full() {
-			let _indent = if is_pretty() {
-				Some(pretty_indent())
-			} else {
-				f.write_char(' ')?;
-				None
-			};
-			write!(f, "PERMISSIONS {}", self.permissions)?;
+			write!(f, " COMMENT {}", comment)?;
 		}
+		let _indent = if is_pretty() {
```

Comment:
Is there any reason we are storing this if we don't need it?

Reply:
Yes, if it's dropped, then the pretty indentation reverts. Hence it's dropped at the end of the formatting function.